### DISCUSSION TITLE
Is there a way to grab the markdown as text?

### DISCUSSION THREAD

**Discussion by altinaaaaaaa** [agusmakmun/django-markdown-editor#261] [Q&A]
In an older project I used [markdownify](https://github.com/erwinmatijsen/django-markdownify) to convert a markdown field into text, which I could then do pythonic things with. For instance in my `models.py` I did

```python
class MyModel(models.Model):
    body = MarkdownxField()

...

    def snip(self):
        return markdownify(self.body)[:150] + "..."
```

to get just the first 150 characters of the body which was written in markdown. Does django-markdown-editor have something like this?

I found that it does have `from martor.utils import markdownify` however, using that on a `MartorField()` does NOT remove the html tags and return only the text. Is there a way to grab the contents of a `MartorField()` stripped of all html formatting?

**Comment by agusmakmun** [comment]
Hello @altinaaaaaaa the official django already have strip tags, so you can use it to remove the html tags instead.

```python
from django.utils.html import strip_tags

html_string = '<strong>Hello</strong>, <p>world!</p>'
stripped_string = strip_tags(html_string)
print(stripped_string)
# Output: Hello, world!
```
